packages/web3-providers/src/alchemy/types.ts

```diff
@@ -0,0 +1,39 @@
+export interface AlchemyResponse {
+    ownedNfts: AlchemyNFT[]
+    pageKey?: string
+}
+
+export interface AlchemyNFT {
+    contract: {
+        address: string
+    }
+    id: {
+        tokenId: number
```

Comment:
Some tokens may have the `tokenId` in string format. A example is: https://opensea.io/assets/ethereum/0x797ce6d5a2e4ba7ed007b01a42f797a050a3cbd8/23210013724799590624788657458228876577121245217617941899843972591016911778602